### ISSUE TITLE
Find a way how to turn HotS Brutal dmg modifier

### ISSUE THREAD

**Issue by Ziktofel** [Ziktofel/Archipelago-SC2-data#310]
From Discord:
>Venat — dnes v 21:51
yeah they're a bit different regardless, not sure if its in scope but I'd love an option to turn that off, hots brutal usually isn't too bad anyway because kerrigan is broken but I hate that mechanic

it's somewhere part of the maps (or a function in a dependency) that does it

**Comment by Ziktofel** [comment]
> BTW with the HotS damage multiplier it looks like it's set on every cinematic exit
(including auto-skipped ones)
```xml
    <CUser id="DifficultySettings">
        <Fields Id="DamageTakenRatio" Type="Fixed" EditorColumn="2"/>
        <Fields>
            <Id value="Name"/>
            <Type value="Text"/>
            <EditorColumn value="1"/>
            <EditorText value="Name"/>
        </Fields>
        <Instances Id="[Default]"/>
        <Instances Id="Casual">
            <Fixed Fixed="0.5">
                <Field Id="DamageTakenRatio"/>
            </Fixed>
            <Text Text="UserData/DifficultySettings/Casual_Name">
                <Field Id="Name"/>
            </Text>
        </Instances>
        <Instances Id="Normal">
            <Fixed Fixed="1">
                <Field Id="DamageTakenRatio"/>
            </Fixed>
            <Text Text="UserData/DifficultySettings/Normal_Name">
                <Field Id="Name"/>
            </Text>
        </Instances>
        <Instances Id="Hard">
            <Fixed Fixed="1">
                <Field Id="DamageTakenRatio"/>
            </Fixed>
            <Text Text="UserData/DifficultySettings/Hard_Name">
                <Field Id="Name"/>
            </Text>
        </Instances>
        <Instances Id="Brutal">
            <Fixed Fixed="1.25">
                <Field Id="DamageTakenRatio"/>
            </Fixed>
            <Text Text="UserData/DifficultySettings/Brutal_Name">
                <Field Id="Name"/>
            </Text>
        </Instances>
    </CUser>
```
> the value is taken from there
The Brutal modifier exists only for HotS and Prologue, and Casual exists for everything that's not WoL (it reduces the enemy health in the missions instead)
Ziktofel — včera v 19:45
The way to deal with this is to set the DamageTakenRatio modifiable (in campaign's UserData.xml) and use the Set User Data action

**Comment by Ziktofel** [comment]
Done
